sdk/core/azure-core-experimental/src/main/java/com/azure/core/experimental/implementation/ARMScopeHelper.java

```diff
@@ -0,0 +1,80 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.core.experimental.implementation;
+
+import com.azure.core.experimental.http.policy.ArmChallengeAuthenticationPolicy;
+import com.azure.core.http.HttpRequest;
+
+import java.util.Locale;
+import java.util.Map;
+
+/**
+ * Defines a few utilities. This is a temporary impl class to support {@link ArmChallengeAuthenticationPolicy} until it
+ * moves to azure-resource-manager package.
+ */
+public final class ARMScopeHelper {
```

Comment:
Why is this called "ARMScopeHelper" if it's helping ARM, Key Vault, etc get their scopes?